app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java

```diff
@@ -70,13 +72,23 @@ public static void showDeleteEntriesDialog(Activity activity, List<VaultEntry> s
         View view = activity.getLayoutInflater().inflate(R.layout.dialog_delete_entry, null);
         TextView textMessage = view.findViewById(R.id.text_message);
         TextView textExplanation = view.findViewById(R.id.text_explanation);
-        String entries = services.stream()
-                .map(entry -> !entry.getIssuer().isEmpty() ? entry.getIssuer()
-                        : !entry.getName().isEmpty() ? entry.getName()
-                        : activity.getString(R.string.unknown_issuer)
-                )
-                .collect(Collectors.joining(", "));
-        textExplanation.setText(activity.getString(R.string.delete_entry_explanation, entries));
+        LinearLayout linearLayout = view.findViewById(R.id.text_entries_layout);
+        for(int i=0;i< services.size();i++) {
+            TextView entry = new TextView(view.getContext());
+            entry.setTextSize(TypedValue.COMPLEX_UNIT_SP, 16);
+            if(!services.get(i).getName().isEmpty() && !services.get(i).getIssuer().isEmpty())
+                entry.setText("\u2022 " + services.get(i).getIssuer() +" \u0028" + services.get(i).getName() + "\u0029");
+            else {
+                if(services.get(i).getName().isEmpty() && services.get(i).getIssuer().isEmpty())
+                    entry.setText(activity.getString(R.string.unknown_issuer));
+                else if(services.get(i).getIssuer().isEmpty())
+                    entry.setText("\u2022 " + services.get(i).getName());
+                else
+                    entry.setText("\u2022 " + services.get(i).getIssuer());
+            }
+            linearLayout.addView(entry);
```

Comment:
Why is this inserting multiple textviews, instead of inserting a line per entry in a single TextView?

Reply:
Actually about this it is not inserting multiple textviews. It is checking if both name and issuer is empty or not.
Case 1 -> Nothing is empty, Execute line number 80 and directly go to line no 89
Case 2 -> Both Name and issuer is empty , Execute line number 83 and directly go to line no 89
Case 3 ->  Issuer is empty , Execute line number 85 and directly go to line no 89
Case 4 ->  Name is empty , Execute line number 87 and directly go to line no 89

Reply:
Right. And on line 89 it inserts a new TextView for every entry.

Reply:
@alexbakker I am sorry but in my country the lock down has completed and I have to go school 10th class so I don't have time to work on android dev anymore 😔

Reply:
No worries. I'll make the requested changes and merge this. Thanks for the help, and good luck in school!

Reply:
Thank you.Hoping to work on aegis soon.